ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
+  ghostAPI:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - Image: registry.hub.docker.com/library/ghost
+          PortMappings:
+            - ContainerPort: 2368
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref ghostAPILogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: logs
+          MountPoints:
+            - 
+              ContainerPath: /var/lib/ghost/content
+              SourceVolume: ghostVolume
+          Name: 0
+          Environment:
+            - Name: url
+              Value: !Join [ '', [ 'http://', !GetAtt Loadbalancer.DNSName ] ]
+            # - Name: database__client
+            #   Value: sqlite3
+            # - Name: database__connection__filename
+            #   Value: '/var/lib/ghost/content/data/ghost-efs.db'
+            - Name: paths__contentPath
+              Value: /var/lib/ghost/content
+            - Name: DB_ARN
+              Value: !Sub arn:aws:rds:${AWS::Region}:${AWS::AccountId}:cluster:${ghostDB}
+            - Name: DB_ROOT_USER_SECRET_ARN
+              Value: !Ref ghostDBRootUserSecret
+            - Name: database__connection__host
+              Value: !GetAtt ghostDB.Endpoint.Address              
+            - Name: database__connection__user
+              Value: root              
+            - Name: database__connection__password
+              Value: !Join ['', ['{{resolve:secretsmanager:', !Ref ghostDBRootUserSecret, ':SecretString:password}}' ]]
+            - Name: database__client
+              Value: mysql
+            - Name: database__pool__min
+              Value: '0'
+            - Name: database__connection__port
+              Value: '3306'
+            - Name: database__connection__database
+              Value: !Ref ghostDB
```

Comment:
This looks like it's supposed to be the name of the database within the database cluster. The value provided here is the identifier of the cluster itself. If you simply need to provide "a" name, I'd hard code it to something like "ghost".